enumerable.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield(self[i])
+      i += 1
+    end
+  end
+
+  def my_each_with_index
+    i = 0
+    while i < length
+      yield(i, self[i])
+      i += 1
+    end
+  end
```

Comment:
* [ ] If no block is given (i.e check with block_given? at the beginning of the method) the method should return an Enumerator.
* [ ] my_each_with_index calls the given block once for each element in self
  - the first argument of my_each should be the item and second argument should be the index. You are doing the opposite.